scalastyle-config.xml

```diff
@@ -86,7 +86,7 @@ This file is divided into 3 sections:
   </check>
 
   <check level="error" class="org.scalastyle.scalariform.ObjectNamesChecker" enabled="true">
-    <parameters><parameter name="regex"><![CDATA[[A-Z][A-Za-z]*]]></parameter></parameters>
+    <parameters><parameter name="regex"><![CDATA[(config|[A-Z][A-Za-z]*)]]></parameter></parameters>
```

Comment:
I'm adding this so that we can have an object named `config`; this seems cleaner than the current approach of having `package object config`, which requires every constant to be tagged as private, instead of just marking the whole object as `private[spark]`.